exchanges/kraken/kraken_test.go

```diff
@@ -2320,3 +2320,79 @@ func TestGetCurrencyTradeURL(t *testing.T) {
 		assert.NotEmpty(t, resp)
 	}
 }
+
```

Comment:
I've been unable to test your changes. I think the culprit is that `kraken_test.go` is not setting API credentials properly:
<img width="1121" alt="image" src="https://github.com/user-attachments/assets/f29ece4f-f26b-41f3-9798-bbc82a9d51db">

Reply:
The issue is that `apiSecret` of `asd` needs to be base64 encoded to `YXNk` since Kraken expects the API secret to be encoded as that string will deactivate auth since it'll throw a warning. Since the logger suppresses output when not initialised, you didn't get the warning outputted (we should rectify this for tests :candle:) . Should get: `Kraken unsuccessful HTTP status code: 500 raw response: {"status":"INTERNAL_SERVER_ERROR","result":"error","errors":[{"code":0,"message":"500 INTERNAL_SERVER_ERROR"}],"serverTime":"2024-07-26T02:35:51.488Z"}, authenticated request failed`

Reply:
Excellent, I'm glad it was my fault 🎉  I've since received this `authenticated request failed EAPI:Invalid key` which is what I wanted